src/modules/did.ts

```diff
@@ -38,14 +38,14 @@ import {
 	DidDocWithMetadata,
 	DidDoc,
 	Metadata
-} from "@canow-co/ts-proto/cheqd/did/v2/index.js"
+} from "@canow-co/canow-proto/dist/cheqd/did/v2"
```

Comment:
It's not a good practice to reference modules directly from `dist`/`build`/`lib` folders.

We should consider adding `main` and `type` (optional in this case, but overall recommended) fields in `package.json` file inside of `@canow-co/canow-proto` repo. For example:
```
...
"main": "dist/index.js",
"types": "dist/index.d.ts",
...
```
This way we will be able to use `dist` folder as a root on module import:
`@canow-co/canow-proto/dist/cheqd/did/v2 -> @canow-co/canow-proto/cheqd/did/v2`

Reply:
Discussed in person, we will need to use something like `exports` feature in TS to make this work due to modules hierarchy from `ts-proto`.

Decided to make this change later in a separate scope/task.